### DISCUSSION TITLE
sqlite keyword  renaming in source code tree ?

### DISCUSSION THREAD

**Discussion by tjyang** [tursodatabase/libsql#38] [General]
I am wondering how forking sqlite will be done exactly ?
Will all the references for upstream sqlite in source tree be changed to libsql ?

**Comment by psarna** [comment]
All original naming should stay - patching it would only make backporting fixes from SQLite tougher. And with all old naming kept intact, it would be easy to distinguish which parts of the code are libSQL-specific

**Reply by tjyang** [reply]
When comes to software packaging stage, how /usr/bin/sqlite3 conflict will be resolved ? if sqlite keyword remain the same ? 
Ex: See RPM spec file at https://github.com/libsql/libsql/blob/main/spec.template
